app/src/main/java/com/example/gdsc_assign/MyPageFragment.kt

```diff
@@ -43,10 +43,18 @@ class MyPageFragment : Fragment() {
                 val intent = Intent(requireContext(), EditActivity::class.java) //액티비티 넘어갈때 사용
                 //                startActivity(intent)
                 intent.putExtra("nickname", tvMyName.text)
-                launcher.launch(intent)
+                mainlauncher.launch(intent)
             }
+        } // editactivity로 넘어가는 부분
+
+        val completetodoObserver = Observer<String> {currentnum ->
+            binding.tvCurrentCompleteTodo.text = "완료 투두 : " + currentnum
+
         }
+        viewModel._currentToDoItem.observe(viewLifecycleOwner,completetodoObserver)
```

Comment:
```suggestion
        viewModel.currentToDoItem.observe(viewLifecycleOwner,completetodoObserver)
```
_currentToDoItem, currentToDoItem 차이가 무엇일까요

Reply:
_currenttodoitem같은 경우 mutable이라 변할수 있지만 currentTodoItem은 못변한다..?관찰자 패턴에서 _currentTodoitem을 변화시키고 그변화를  currentTodoitem에 적용시킨 후에 그걸 관찰한다!